docs/devupdate.md

```diff
@@ -40,6 +40,30 @@ will only update the second enrol instance.
 
 A new method `enrol_plugin:find_instance()` is added to the enrol plugin interface to allow plugins to find an existing instance of the same type in the course. If you want your enrolment method to be supported in CSV course upload, you need to implement this [method](./apis/plugintypes/enrol#enrol_pluginfind_instance-stdclass)
 
+## Bootstrap preparations for version 5
+
+<Since version="4.4" issueNumber="MDL-71979" />
+
+Some of the Bootstrap 4 classes will deprecated or dropped in its version 5. To prepare for this, some of the current Bootstrap 4 classes usages have been replaced with version 5 compatible classes. This will help us to upgrade to Bootstrap 5 in the future.
+
+- Badge color class helpers `.badge-[warning|info|...]` have been replaced with `.bg-warning` `.bg-info`, ... and the corresponding text color classes `.text-dark` `.text-white` to meet accessibility contrast.
```

Comment:
This whole section is confusing. Shoudl we be using `bg-warning` with `text-dark`?

Reply:
You are totally right, it was super-confusing. I've re-written it

Reply:
The new version was based the in the BS5 migration documentation https://getbootstrap.com/docs/5.0/migration/#badges